src/tracing/logger/Logger.ts

```diff
@@ -13,6 +18,8 @@ export type LoggerOptions = {
     debug?: boolean;
 }
 
+const FLUSH_INTERVAL_MS = 5 * 1000; // 5 seconds
```

Comment:
Make this env var?

Reply:
I'm not 100% on this but I don't think `env` vars are good DX on a _user's_ machine.

Instead, we could add these as configuration options inside of `ollyllm.config.json` which we already need for tests.